pdal/Reader.cpp

```diff
@@ -43,7 +43,26 @@ void Reader::readerAddArgs(ProgramArgs& args)
     m_filenameArg = &args.add("filename", "Name of file to read", m_filename);
     m_countArg = &args.add("count", "Maximum number of points read", m_count,
         (std::numeric_limits<point_count_t>::max)());
-    addSpatialReferenceArg(args);
+
+    args.add("override_srs", "Spatial reference to apply to data",
+            m_overrideSrsArg);
+
+    args.addSynonym("override_srs", "spatialreference");
+
+    args.add("default_srs",
+            "Spatial reference to apply to data if one cannot be inferred",
+            m_defaultSrsArg);
+}
+
+void Reader::readerInitialize(PointTableRef)
+{
+    if (!m_overrideSrsArg.empty() && !m_defaultSrsArg.empty())
```

Comment:
I tend to agree, initially I wanted to add `SpatialReference::exists` which was the negation of empty for these checks and then I noticed `valid()`.  But since the implementation of `valid()` isn't quite equivalent to `!empty()` I was playing it safe with the current method.  I will switch them.

Reply:
Would this suggested behavior still work properly if an existing but non-valid `default_srs` or `override_srs` was supplied?